packages/renderer/src/lib/container/ContainerActions.svelte

```diff
@@ -41,51 +42,93 @@ function openGenerateKube(): void {
 function deployToKubernetes(): void {
   router.goto(`/deploy-to-kube/${container.id}/${container.engineId}`);
 }
+
+// Below is the fuctionality for the "kebab" menu
+
+// Show and hide the menu using clickOutside
+let showMenu = false;
+
+// If we touch outside the window, hide the menu
+let outsideWindow;
+
+// If we hit ESC while the menu is open, close it
+function handleEscape({ key }) {
+  if (key === 'Escape') {
+    showMenu = false;
+  }
+}
+
+function toggleMenu() {
+  showMenu = !showMenu;
+}
+
+// If we click outside the menu, close the menu
+function onWindowClick(e) {
+  if (outsideWindow.contains(e.target) == false) showMenu = false;
+}
```

Comment:
I think the toggle should be handle by a <KebabMenu> component , so not part of this svelte component